data/render/src/bin/main.rs

```diff
@@ -1,13 +1,13 @@
-use askama::Template;
 use anyhow::{Context, Result};
```

Comment:
Can't say much about the Rust part here, due to missing knowledge of Rust itself.
Skipping this one.